Preprocessing/tst_data_quality_control.py

```diff
@@ -0,0 +1,216 @@
+"""
+Validate data quality in TST experiment folders
+  + check XLS metadata file and show :
+    + ERROR when file is not found
+
+  + check CSV TST files and show :
+    + ERROR when :
+      + mandatory column is not found
+      + unexpected type is found in a column
+    + WARNING when :
+      + unexpected column found
+"""
+
+#
+# TODO :
+# + check JSON files (from XLS)
```

Comment:
Is it okay ? to have a. todo ?

Reply:
Ths file doesn't exist anymore in the latest commit